First/ViewController.swift

```diff
@@ -77,11 +73,20 @@ class ViewController: UIViewController {
         legalTermsLabel.translatesAutoresizingMaskIntoConstraints = false
         legalTermsLabel.textAlignment = .center
         legalTermsLabel.textColor = .white
-        legalTermsLabel.text = "This space can be used for any legal terms."
+        legalTermsLabel.text = legalString
         view.addSubview(legalTermsLabel)
     }
     
     func setupConstraints(){
+        let kLogoWidth: CGFloat = 250
+        let kLogoHeight: CGFloat = 100
+        let kLeftMargin: CGFloat = 10
+        let kRightMargin: CGFloat = -10
+        let kTopMargin: CGFloat = 10
+        let kBlackBottomMargin: CGFloat = -50
+        let kTroubleBottomMargin: CGFloat = -20
+        let kElementsHeight: CGFloat = 50
+        let kLegalTermsHeight: CGFloat = 20
```

Comment:
To było w dobrym miejscu wsadzone wcześniej, w sensie na początku deklaracji klasy. Chodziło mi o to żeby zadeklarowane były tylko dodatnie liczby, a przy użyciu dopiero dawać -. 
Na przykład _button.rightAnchor.constraint(equalToConstant: -kRightMargin_)